command/src/config.rs

```diff
@@ -1144,8 +1144,14 @@ fn default_accept_queue_timeout() -> u32 {
 }
 
 impl Config {
-    pub fn load_from_path(path: &str) -> io::Result<Config> {
-        FileConfig::load_from_path(path).map(|config| config.into(path))
+    pub fn load_from_path(path: &str) -> anyhow::Result<Config> {
+        let mut config = FileConfig::load_from_path(path).map(|config| config.into(path))?;
+
+        config.saved_state = config
```

Comment:
Is the `saved_state_path` method is used somewhere else ? If it is not the case, it could be a private method

Reply:
it is used only once, here. Will do.